Remove statement that Symfony Reverse Proxy is required when setting TrustedProxies (#1961)
ezsystems/developer-documentation: docs/guide/cache/symfony_reverse_proxy.md

### Configure Symfony front controller

> You need to consider your `TrustedProxy` configuration when you use Symfony [behind a load balancer or a reverse proxy](https://symfony.com/doc/5.1/deployment/proxies.html)

Reviewer: ```suggestion
You need to consider your `TrustedProxy` configuration when you use Symfony [behind a load balancer or a reverse proxy.](https://symfony.com/doc/5.1/deployment/proxies.html)
```

> You need to consider your `TrustedProxy` configuration before you configure Symfony to [work behind a load balancer or a reverse proxy](https://symfony.com/doc/5.1/deployment/proxies.html)

Author: ```suggestion
You need to consider your `TrustedProxy` configuration when you use Symfony [behind a load balancer or a reverse proxy](https://symfony.com/doc/5.1/deployment/proxies.html)
```
